pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -882,12 +882,21 @@ type ClusterConfig struct {
 	//       value: '"wlzjyw.bei2zfylhs2by0wd"'
 	BootstrapToken string `yaml:"token,omitempty"`
 	//   description: |
-	//     The key used for the [encryption of secret data at rest](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/).
+	//     A key used for the [encryption of secret data at rest](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/).
 	//   examples:
 	//     - name: Decryption secret example (do not use in production!).
+	//     Enables encryption with AESCBC.
 	//       value: '"z01mye6j16bspJYtTB/5SFX8j7Ph4JXxM2Xuu4vsBPM="'
 	ClusterAESCBCEncryptionSecret string `yaml:"aescbcEncryptionSecret"`
```

Comment:
```suggestion
	ClusterAESCBCEncryptionSecret string `yaml:"aescbcEncryptionSecret,omitempty"`
```

we should make both `omitempty`, one for aesthetic reasons, but the new one should be omitempty always, as otherwise the config generate for Talos <= 1.2 will be broken due to unknown YAML key